### ISSUE TITLE
ERROR 400 when uploading a local backup 

### ISSUE THREAD

**Issue by RenzoBeux** [Sebclem/hassio-nextcloud-backup#44]
**Describe the bug**
```
[2021-01-18 02:04:15] [error]: 	Fail to upload backup to home assitant (HTTPError: Response code 400 (Bad Request)) !
(node:377) UnhandledPromiseRejectionWarning: Fail to upload backup to home assitant (HTTPError: Response code 400 (Bad Request)) !
(node:377) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:377) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**To Reproduce**
Upload a local backup

**Expected behavior**
It should backup sucessfuly

**System information**
- Home Assistant Version : latest
- Home Assistant Supervisor Version : latest
- Nextcloud Version : latest
- Addon version : latest

**Comment by Sebclem** [comment]
Hi, take a look a this comment: https://github.com/Sebclem/hassio-nextcloud-backup/issues/27#issuecomment-709603344
Someone have the same issue and it's related to trusted domain config in nextcloud.
You need to configure trusted domain in config.php:
- edit `/var/www/nextcloud/config/config.php`
- Add or edit your trusted domain, this may look like this:
```php
'trusted_domains' =>
  array (
    0 => '192.168.0.29',
    1 => 'cloud.example.com',
  ),
```

**Comment by RenzoBeux** [comment]
Hi! I already set my trusted domains, i can even accces on the web browser.

**Comment by Sebclem** [comment]
The domain that you use on the addon config is present in trusted domain ?
If yes, do you have something in nextcloud logs when the upload fail ?

**Comment by RenzoBeux** [comment]
Yes indeed. What i do see strange (but maybe just a frontend bug) is when i press upload 3 mensajes on left appear.
![image](https://user-images.githubusercontent.com/47984043/104920534-ae498980-5976-11eb-9c6b-bdc5983ef46e.png)
![image](https://user-images.githubusercontent.com/47984043/104920575-bbff0f00-5976-11eb-93a0-ea6c645b9a38.png)

**Comment by RenzoBeux** [comment]
Yep , i should check the logs before. Sorry im pretty noobie.
![image](https://user-images.githubusercontent.com/47984043/104920988-742cb780-5977-11eb-80e7-bbfd3d8f29e5.png)

Now i would like why it failed. Because if i write the password wrong in the Nextcloud config an 401 error appear, so i have the right credentials in the setting. The problem arises when trying to upload.

**Comment by RenzoBeux** [comment]
Sorry my mistake (deleted a comment). Nothing appear on logs when upload is tried.

**Comment by Sebclem** [comment]
Ok i think i have found the issue, can you check with the last release ?

**Comment by RenzoBeux** [comment]
Oh god!!!! Working now!! Thank you very much!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
